re.rizin.cutter.yml

```diff
@@ -0,0 +1,43 @@
+app-id: re.rizin.cutter
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: cutter
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+
+modules:
+  - name: rizin
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - --prefix=/app
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/rizin/releases/download/v0.2.0/rizin-src-v0.2.0.tar.xz
+        sha256: 0861dea36e3bb3285afab56d880433d1790460af69e031d39cb2b616104ebc4f
+
+  - name: cutter
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCUTTER_USE_BUNDLED_RIZIN=off
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCUTTER_ENABLE_KSYNTAXHIGHLIGHTING=on
+      - -DCUTTER_ENABLE_PYTHON=on
+      - -DCUTTER_EXTRA_PLUGIN_DIRS=/app/share/rizin/cutter/plugins\;
+#      - -DCUTTER_ENABLE_PYTHON_BINDINGS=on
```

Comment:
CUTTER_ENABLE_PYTHON=on without CUTTER_ENABLE_PYTHON_BINDINGS=on is somewhat useless. Does org.kde.Platform runtime not include pyside and shiboken ?

Reply:
Well cmake can't find it and a quick "find" did not find them so they need to be added.

Reply:
Looks like that's what other projects also do https://github.com/search?q=org%3Aflathub+PySide2&type=code

Reply:
no, it doesn't include either.

Reply:
I've tried a few things to make pyside2/shiboken2 work:
- using wheel packages
- using the method in the package timecard
- using the method in the package FreeCAD

The wheel method does not seem to install the .cmake needed. As for the other methods it compiles but the cutter compilation breaks : https://pastebin.com/i7xFP45Z

The issue does not come from the versions of Qt/shiboken as they are the same for the Cutter AppImage.

I'm still investigating the issue.

Reply:
The issue was the same as for the pyside build. The shibokken binary does not use system libraries (it rejects it) so you need to give it another way. It is fixed